indico/modules/rb/operations/bookings_test.py

```diff
@@ -53,14 +53,19 @@ def past_booking_occurrences_are_cancelled(dummy_user, create_reservation):
     start_dt = datetime.today().replace(hour=8, minutes=30) - timedelta(days=2)
     end_dt = datetime.today().replace(hour=17, minutes=30) + timedelta(days=4)
     reservation = create_reservation(start_dt=start_dt, end_dt=end_dt, repeat_frequency=RepeatFrequency.DAY)
-    new_reservation = split_booking(reservation, {
```

Comment:
Oh fun, this unit test is completely broken and has never been executed (opened #6210 for it since I don't feel like touching that...). Anyway your change in here looks fine.